data/projects.js

```diff
@@ -0,0 +1,101 @@
+},
+
+{
+  projectName: 'ROCK - PAPER - SCISSORS',
+  studentName: 'Diana Colombiana',
+  presentationLink: 'https://slides.com/dianablanco/deck/#/',
+  projectLink: 'https://dbpautt.github.io/Rock-paper-scissors/',
+  imageURL: 'https://media.giphy.com/media/2t9m7p4fY2DQEKT7fM/giphy.gif'
+},
+
+{
+  projectName: 'Franguinho',
+  studentName: 'Caroline',
+  presentationLink: 'https://docs.google.com/presentation/d/1jOY2LA9R1Gi95OuXFTCMzi3nCLeBuWuUaXtUCPO_lzc/edit#slide=id.p',
+  projectLink: 'https://caroline-gmr.github.io/franguinho/',
+  imageURL: 'https://media.giphy.com/media/9JegwSqBdhZPTWZTTU/giphy.gif'
+},
+
+{
+  projectName: 'Gay Heaven',
+  studentName: 'Barbara',
+  presentationLink: 'https://docs.google.com/presentation/d/1gNDbHGPemgxaYoJ2x4el1SlYgUC3Qwr-Gb6QCHLFcd8/edit',
+  projectLink: 'https://barkrrr.github.io/gay-heaven/',
+  imageURL: 'https://media.giphy.com/media/cRNb3CMoMWMh0BOvX2/giphy.gif'
+},
+
+{
+  projectName: 'Brick Breaker',
+  studentName: 'Jonathan',
+  presentationLink: 'https://slides.com/jonathanskudlik/deck#/',
+  projectLink: 'https://jsplmns.github.io/IRNHCK_m1-project-brick-breaker/',
+  imageURL: 'https://media.giphy.com/media/1wpxKB39vRX6xMaIY2/giphy.gif'
+},
+
+{
+  projectName: 'Brick Race',
+  studentName: 'Yenderly',
+  presentationLink: 'https://slides.com/yendy/my-name-is-yenderly-and-i-present-you-brick-race#/',
+  projectLink: 'https://slides.com/yendy/my-name-is-yenderly-and-i-present-you-brick-race#/',
+  imageURL: 'https://media.giphy.com/media/cYHxcdV6qYuAmEYA65/giphy.gif'
+},
+
+{
+  projectName: 'Pong Clone',
+  studentName: 'Gabriela',
+  presentationLink: 'https://docs.google.com/presentation/d/1UF--GqueyfpnKf6giEiNEeotXsQCD-MplbPjlLxGcoE/edit?usp=sharing',
+  projectLink: 'https://gap1885.github.io/Pong-clon-/',
+  imageURL: 'https://media.giphy.com/media/AgORULDNGG4L06s0qz/giphy.gif'
+},
+
+{
+  projectName: 'Web Dev life',
+  studentName: 'Francesca',
+  presentationLink: 'https://docs.google.com/presentation/d/1dgAmdQohaIMBzYpxkBA0fZXbUOYnzNlT29QZfzKbjzc/edit?usp=sharing',
+  projectLink: 'https://francescaropolo.github.io/webdev-life-game/',
+  imageURL: 'https://media.giphy.com/media/xV8c4uqtlfX5Z2SKKe/giphy.gif'
+},
+
+{
+  projectName: 'Underdog',
+  studentName: 'Axel',
+  presentationLink: 'https://slides.com/axelgarcia-1/project-1underdog#/',
+  projectLink: 'https://axelgar.github.io/underdog/',
+  imageURL: 'https://media.giphy.com/media/3aW3JtgEEYY6Hd5IhZ/giphy.gif'
+}
+
+];
+
+mongoose.connect(process.env.MONGODB_URI, {
+  keepAlive: true,
+  useNewUrlParser: true,
+  reconnectTries: Number.MAX_VALUE
+})
+  .then(() => {
+    return Project.remove({});
+  })
+  .then(() => {
+    return Project.insertMany(projects);
+  })
+  .then((result) => {
+    console.log('successfully added to database', result);
+    mongoose.connection.close();
+  })
+
+  .catch((error) => {
+    console.log('there has been an error', error);
+  });
```

Comment:
just the `const data = ...'

and module.exports = data

and then the actual seed script should go inside `bin/seed/projects.js`